ick/config/rule_repo.py

```diff
@@ -45,10 +46,10 @@ def discover_rules(rtc: RuntimeConfig) -> Sequence[RuleConfig]:
 
 
 @ktrace("mount.url", "mount.path")
-def load_rule_repo(mount: Mount) -> RuleRepoConfig:
+def load_rule_repo(mount: Mount, skip_update=False) -> RuleRepoConfig:
```

Comment:
```suggestion
def load_rule_repo(mount: Mount, *, skip_update=False) -> RuleRepoConfig:
```
This makes it so that `skip_update` has to be provided as a keyword, helping to avoid mismatches when there are more parameters.

Reply:
Ideally, we'd add this to `update_local_cache` also, especially since it has two boolean arguments that would be easy to get in the wrong order.